CarRecognition/Source Files/Common/Services/CarClassificationService.swift

```diff
@@ -76,7 +76,8 @@ internal final class CarClassificationService {
         else {
             return
         }
-        let recognitionResult = classifications.compactMap { RecognitionResult(label: $0.identifier, confidence: $0.confidence, carsDataService: carsDataService) }
+        let recognitionResult = classifications.filter { $0.confidence > Constants.Recognition.Threshold.minimum }
```

Comment:
This indentation looks wrong. I think it should be something like this:
```
        let recognitionResult = classifications
            .filter { $0.confidence > Constants.Recognition.Threshold.minimum }
            .compactMap { RecognitionResult(label: $0.identifier, confidence: $0.confidence, carsDataService: carsDataService) }
```

Reply:
Yes, I believe you're right! Thanks!

Reply:
But honestly, it is right according to the Style Guide but for me the option which I used is more readable :P 
Anyway, I'll change it to conform to the guide.